skore/src/skore/item/cross_validation_item.py

```diff
@@ -288,3 +373,68 @@ def factory(
     def plot(self):
         """A plot of the cross-validation results."""
         return plotly.io.from_json(self.plot_bytes.decode("utf-8"))
+
+
+class AggCrossValidationItem(Item):
```

Comment:
This naming is buzzing me. We do not use abbreviations that much why not something explicit ?
Maybe `CrossValidationsAggregationItem` ☕ style indeed but explicit.

Reply:
Done!